lib/pacemaker/libpacemaker_private.h

```diff
@@ -859,6 +859,11 @@ void pcmk__output_bundle_actions(pe_resource_t *rsc);
 G_GNUC_INTERNAL
 void pcmk__bundle_add_actions_to_graph(pe_resource_t *rsc);
 
+G_GNUC_INTERNAL
+void pcmk__bundle_add_utilization(const pe_resource_t *rsc,
```

Comment:
This method is missing doxygen. No pressure to add it now.

Reply:
I went back and forth over whether method implementations should have a copy of the method doxygen, or just a one-line reference to it. Still not sure which is better.